src/conf_mode/dns_dynamic.py

```diff
@@ -88,7 +89,12 @@ def verify(dyndns):
         # If dyndns address is an interface, ensure that it exists
         # and that web-options are not set
         if config['address'] != 'web':
-            verify_interface_exists(config['address'])
+            # exclude check interface for dynamic interfaces
+            interface_filter = ('ppp')
```

Comment:
Please tighten the prefix from ppp to pppoe and also add sstpc.

Reply:
pppoe  starts with `ppp` :)

Reply:
Added `sstpc`